DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -37,6 +37,8 @@
  */
 class DoctrineMongoDBExtension extends AbstractDoctrineExtension
 {
+    public const CONFIGURATION_TAG = 'doctrine.odm.configuration';
```

Comment:
doctrine bundle defines this tag in 2 places, as a private const in the compatibilitypass and as a public const in another pass. Imo it makes most sense to define it in 1 place; the extension class, but maybe someone can elaborate on why doctrine bundle implemented it in that way.

Reply:
Agree. Only change I would make is to mark the constant as internal, since we don't consider this part of the public API.
```suggestion
    /** @internal */
    public const CONFIGURATION_TAG = 'doctrine.odm.configuration';
```

Reply:
I thought the entire class already marked `@internal`, but I see now that it isn't. Perhaps we should consider marking the class internal instead?

Reply:
Yes, that makes more sense.

Reply:
After discussion, decided that because bundles are allowed to load other bundles' configuration to allow inter-bundle integration, to make just the const internal.